src/bufr/ParserFactory.h

```diff
@@ -0,0 +1,83 @@
+/*
+ * (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+ *
+ * This software is licensed under the terms of the Apache Licence Version 2.0
+ * which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+#pragma once
+
+#include <map>
+#include <memory>
+#include <string>
+#include <ostream>
+
+#include "eckit/config/LocalConfiguration.h"
+#include "eckit/exception/Exceptions.h"
+
+#include "Parser.h"
+
+static const char* PARSER_NAME = "name";
```

Comment:
Move inside the functions. Otherwise, all code that includes this header file adds in the PARSER_NAME symbol.